odoo/tools/image.py

```diff
@@ -127,7 +127,7 @@ def image_base64(self, quality=0, output_format=''):
             output_format = 'JPEG'
 
         if not self.operationsCount and output_format == self.original_format and not quality:
-            return self.base64_source
+            return self.image
```

Comment:
This diff is not correct, `self.image` is the PIL class, not the bytes, and the method is supposed to return the original bytes.

Error from the caller when reaching this case:
`TypeError: a bytes-like object is required, not 'PngImageFile'`